subsys/bluetooth/controller/Kconfig.ll_sw_split

```diff
@@ -598,12 +598,29 @@ config BT_CTLR_DYNAMIC_INTERRUPTS
 	  permit use of SoC's peripheral for custom use when Bluetooth is not
 	  enabled.
 
+choice
+	prompt "Optimazation options"
+	default BT_CTLR_OPTIMIZE_FOR_SPEED
```

Comment:
Should we default to `BT_CTLR_LTO` if `LTO=y`?

Reply:
Moved out `BT_CTLR_LTO` from the `choice`  to make LTO and compiler optimization mutually exclusive.